Modules/_functoolsmodule.c

```diff
@@ -703,7 +703,7 @@ functools_reduce(PyObject *self, PyObject *args)
             // bpo-42536: The GC may have untracked this args tuple. Since we're
             // recycling it, make sure it's tracked again:
             if (!_PyObject_GC_IS_TRACKED(args)) {
-                _PyObject_GC_TRACK(args);
+                PyObject_GC_Track(args);
```

Comment:
Did you mean for this to be part of this PR?

Reply:
Yes, _PyObject_GC_TRACK is an inline function which uses `_py_current_interpreter`, so cannot be used in modules.
I could modify `_PyObject_GC_TRACK` but that would impact the performance of core code.

Reply:
Thanks for pointing that out.  I thought I'd checked for that but clearly I missed it. 🙂